### ISSUE TITLE
the padding in maxpooling of the vgg16.py

### ISSUE THREAD

**Issue by zhangyingbit** [machrisaa/tensorflow-vgg#68]
why you use padding="SAME" for the maxpooling?  I think there should be padding="VALID"
```
 def avg_pool(self, bottom, name):
        return tf.nn.avg_pool(bottom, ksize=[1, 2, 2, 1], strides=[1, 2, 2, 1], padding='SAME', name=name)
```

**Comment by zhangyingbit** [comment]
There padding="SAME is equal to padding="VALID"
